src/main/java/com/iridium/iridiumteams/listeners/SpawnerSpawnListener.java

```diff
@@ -28,8 +28,11 @@ public void onCreatureSpawn(SpawnerSpawnEvent event) {
             if (!teamEnhancement.isActive(spawnerEnhancement.type)) return;
             if (data == null) return;
 
+            int spawnCount = spawner.getSpawnCount();
             spawner.setSpawnCount((spawner.getSpawnCount() * data.spawnMultiplier) + data.spawnCount);
+            if(spawner.getMaxSpawnDelay() != data.spawnDelay) { spawner.setMaxSpawnDelay(data.spawnDelay); }
             spawner.update(true);
```

Comment:
What if its not active, do we need to set it back to the default?

What is the default?